db/migrate/20170911102046_add_missing_container_networking_pillars.rb

```diff
@@ -0,0 +1,23 @@
+class AddMissingContainerNetworkingPillars < ActiveRecord::Migration
+  def up
+    Pillar.find_or_create_by pillar: :cluster_cidr do |pillar|
+      pillar.value = "172.20.0.0/16"
+    end
+    Pillar.find_or_create_by pillar: :cluster_cidr_min do |pillar|
+      pillar.value = "172.20.50.0"
+    end
+    Pillar.find_or_create_by pillar: :cluster_cidr_max do |pillar|
+      pillar.value = "172.20.199.0"
+    end
+    Pillar.find_or_create_by pillar: :cluster_cidr_len do |pillar|
+      pillar.value = "24"
+    end
+    Pillar.find_or_create_by pillar: :services_cidr do |pillar|
+      pillar.value = "172.21.0.0/16"
```

Comment:
Oh - we'll need to add dns:cluster_ip here too, as that needs to be inside the services CIDR

Reply:
Also, what about `api:cluster_ip` ?